backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/translations/v2TranslationsController/TranslationsControllerViewTest.kt

```diff
@@ -245,6 +246,24 @@ class TranslationsControllerViewTest : ProjectAuthControllerTest("/v2/projects/"
       }
   }
 
+  @ProjectJWTAuthTestMethod
+  @Test
+  fun `returns keys filtered by provided tags`() {
+    testData.generateLotOfData()
+    testDataService.saveTestData(testData.root)
+    userAccount = testData.user
+    val expectedKeys = IntRange(1, 9).map { "key 0$it" }
```

Comment:
In kotlin, you can define range this way `(1..9)`.

However, it's not very clean to rely on the range. What if someone later comes there and edit the test data like you did.

Reply:
Now deleted.